andela/ah-django-unchained-frontend
# #162948808 Create a comment

#### What does this PR do?
Creates a user Interface through which a user creates a comment for an article that is published


#### Description of Task to be completed?
* [x]   Make changes to textarea component
* [x]   Create a comment for an article that is published
* [x]   Add actions and types for article comment
* [x]   Add reducer for article comment
* [x]   Consume the `post comment` API endpoint
* []   Write Tests

#### How should this be manually tested?
1. Clone this repository on your local machine using
   `git clone https://github.com/andela/ah-django-unchained-frontend.git`
2. Change your working directory to the project's directory using
   `cd ah-django-unchained-frontend`
3. Change to the specific branch `ft-create-draft-article-162948806 ` using
   `git checkout ft-create-comment-162948808 `
4. Create a `.env` file and inside it add this line `REACT_APP_BASE_URL=app-base-url` where `app-base-url ` is the root url of the backend API
5. Run `npm install` to install all the modules listed in the `package.json` file.
6. Run `npm start` command to start the application.
7. The application should start successfully. On the browser, navigate to the URL `http://localhost:3000/createArticle`,  and select an article you want to comment on.

Screenshot
![image](https://user-images.githubusercontent.com/18655501/53624411-43232c80-3c11-11e9-840a-326b1a8ad0af.png)





[Starts #162948807](https://www.pivotaltracker.com/story/show/162948808)
